lleida_net/click_sign/serializers.py

```diff
@@ -135,6 +135,49 @@ def create_model(self, data):
         return Objectify(**data)
 
 
+class SignatoryStampSchema(Schema):
+    signatory_id = fields.Integer(required=True)
+    signature_id = fields.Str(required=True)  # Api defines this as String...
+    file_group = fields.Str(required=True)
+
+    @post_load
+    def create_model(self, data):
+        return Objectify(**data)
+
+
+class GetDocument(Schema):
+    content = fields.Str(required=True)
+    filename = fields.Str(required=True)
+    file_url = fields.Str(required=True)
+
+    @post_load
+    def create_model(self, data):
+        return Objectify(**data)
+
+
+class APIResponseGetDocument(Schema):
+    file = fields.List(fields.Nested(GetDocument), many=True, required=True)
```

Comment:
Be careful, `file` is a Python `type`. Please, also review other exiting `file` definitions in other schemas.